src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -1036,7 +1038,12 @@ public String toString() {
       } else if (from == null || from.equals("")) {
         return to;
       } else if (readOnly == null || !readOnly) {
-        return from + ":" + to;
+          if (noCopy == null || !noCopy) {
```

Comment:
Are `noCopy` and `readOnly` mutually exclusive? As in, you can only ever have `from:to:ro` or `from:to:nocopy` but never `from:to:ro:nocopy`?